src/hooks/useMixpanel.ts

```diff
@@ -834,6 +836,27 @@ export default function useMixpanel(currencies?: { [field in Field]?: Currency }
           mixpanel.track('Bridge - User click to Subscribe button')
           break
         }
+        case MIXPANEL_TYPE.BRIDGE_CLICK_DISCLAIMER: {
+          if (typeof payload !== 'boolean') {
+            throw new Error(`Wrong payload type for Mixpanel event: ${MIXPANEL_TYPE.BRIDGE_CLICK_DISCLAIMER}`)
```

Comment:
will crash or not, double check

Reply:
Will crash in dev mode only. This works like a type hint for what payload should be